<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeWindow</class>
 <widget class="QMainWindow" name="EncodeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Steganograf</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="rLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Czerwony:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gLabel">
             <property name="text">
              <string>Zielony:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bLabel">
             <property name="text">
              <string>Niebieski:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSlider" name="rSlider">
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="sliderPosition">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="gSlider">
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="bSlider">
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="rValLabel">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gValLabel">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bValLabel">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="bitNumbersCheckBox">
       <property name="text">
        <string>Numery bitów w składowych</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hideButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ukryj</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="selectTargetButton">
           <property name="text">
            <string>Wybierz nośnik</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectSourceButton">
           <property name="text">
            <string>Wybierz plik ukrywany</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="targetFilenameLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sourceFilenameLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="amountLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="usedSpaceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showGraphWindowButton">
         <property name="text">
          <string>Pokaż wykres zajętego miejsca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewPicturesButton">
         <property name="text">
          <string>Pokaż obrazy przed i po zakodowaniu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNowy">
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
  <action name="actionZako_cz_2">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>selectTargetButton</tabstop>
  <tabstop>selectSourceButton</tabstop>
  <tabstop>rSlider</tabstop>
  <tabstop>gSlider</tabstop>
  <tabstop>bSlider</tabstop>
  <tabstop>bitNumbersCheckBox</tabstop>
  <tabstop>hideButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionZako_cz_2</sender>
   <signal>triggered()</signal>
   <receiver>EncodeWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
